### DISCUSSION TITLE
.env format

### DISCUSSION THREAD

**Discussion by adman120** [firefly-iii/firefly-iii#5915] [Firefly III]
Can I replace the equals with : in the env file? I have preset variables that are already in place and do not want to have to update them in multiple places. an example being replacing  TZ=$XXX with TZ: $XXX? Thank you.

EDIT::
I saw "If you use Docker or similar, you can set these variables from a file by appending them with _FILE" what does that mean?

**Comment by JC5** [comment]
No, you have to stick to the format. 

In regards to the _FILE environment variables: you can do something like APP_KEY_FILE=/path/to/appkey.txt. This is useful in some docker environments.
